lib/primer/open_project/forms/rich_text_area.rb

```diff
@@ -17,6 +17,14 @@ def initialize(input:, rich_text_options:)
           @rich_text_options = rich_text_options
           @text_area_id = rich_text_options.delete(:text_area_id) || builder.field_id(@input.name)
         end
+
+        private
+
+        def rich_text_options
+          @rich_text_options.tap do |options|
+            options[:textAreaAriaLabel] = options.delete(:aria_label) if options.key?(:aria_label)
+          end
+        end
```

Comment:
we need to process various `rich_text_options` before being passed as `inputs:` to `angular_component_tag`. Let's either do the Hash munging in the ERb template OR in `rich_text_area.rb`, but not in both.

e.g. 

```rb
          @rich_text_options = rich_text_options
          @text_area_id = rich_text_options.delete(:text_area_id) || builder.field_id(@input.name)
          @text_area_aria_label = rich_text_options.delete(:aria_label) if rich_text_options.key?(:aria_label)
```

Reply:
Agree, considered this but didn't have better ideas at the time :)